ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/ethernet-interfaces.md

```diff
@@ -1156,6 +1199,96 @@
                 #   - 1-10
               - range: <str; required; unique>
                 mac_address_maximum: <int>
+          tap:
+
+            # VLAN ID or range of VLAN IDs within range 1-4094.
+            allowed_vlan: <str>
+            default:
+
+              # Tap group names for the interface.
+              groups:
+                - <str>
+
+              # Interfaces like -  Ethernet, InternalRecirc, Port-Channel, Recirc-Channel.
+              interfaces:
+                - <str>
+
+              # Default nexthop-group names.
+              nexthop_groups:
+                - <str>
+            identity:
+
+              # Tap port ID tag.
+              port_id: <int; 1-4094>
+
+              # Inner tap port ID tag.
+              inner_port_id: <int; 1-4094>
+
+            # Pop all MPLS labels.
+            mpls_pop_all: <bool>
+
+            # Native VLAN ID when interface is in tap mode.
+            native_vlan: <int; 1-4094>
+            truncation:
+              enabled: <bool; required>
+
+              # Ingress packet truncation size in bytes.
+              size: <int; 100-9236>
+            mac_address:
+
+              # MAC address for the source.
+              src: <str>
+
+              # MAC address for the destination.
+              dest: <str; required>
```

Comment:
better we can add a regex on input for these mac address as it can take in format of H.H.H but in running config it change and we should match the running config.
for example for command `switchport tap mac-address dest 11.11.11 src 22.22.22` it convert into `switchport tap mac-address dest 00:11:00:11:00:11 src 00:22:00:22:00:22`

Reply:
Added